### PR TITLE
Please help: HTML in helper for stars not working

### PR DESCRIPTION
**Pull request by adaau**
@f3r Something funny with the html in playspaces_helper. Can you please take a look?

### CODE REVIEW

File: app/helpers/playspaces_helper.rb [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+  end
+
+  def avg_rating(playspace)
+    if playspace.reviews.count > 0 && playspace.reviews.average(:rating) >= 0
+      html = "<span>"
+      playspace.reviews.average(:rating).floor.times do
+        html += '<span class="glyphicon glyphicon-star yellow"></span>'
+      end
+      html += "</span>"
+      html
```
Comment (f3r) [inline app/helpers/playspaces_helper.rb]:
```rb
html.html_safe
```

File: app/views/reviews/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>Edit review</h1>
```
Comment (dshcheung) [inline app/views/reviews/edit.html.erb]: please delete this file if you are not using it. 15 push ups

File: app/views/reviews/new.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1>New review</h1>
```
Comment (dshcheung) [inline app/views/reviews/new.html.erb]: remove if not used. another 15

File: app/views/shared/_rating.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<% rating = playspace.reviews.average(:rating) %>
+<span>
+  <% rating.floor.times do %>
+    <span class="glyphicon glyphicon-star yellow"></span>
```
Comment (dshcheung) [inline app/views/shared/_rating.html.erb]: since we have the helper we don't need this right? another 15

**Comment by dshcheung** [comment]
also. please look at the file changes tab and add a space at the end of the file for those that needs it

**Comment by dshcheung** [comment]
can merge when changes are complete

**Pull request merged**